recipes/sqlitecpp/all/patches/3.1.1/0001-cmake-add-omit-load-extensions.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
I just noticed this is back porting a feature you added https://github.com/SRombauts/SQLiteCpp/commit/925923c50773a69135f041a4b79436eb09b65007

Reply:
Sure. I think it will release a new version in future.